telosnetwork/open-block-explorer
# recover resources delegated to other accounts

# Fixes #504

## Description
Until now we didn't provide a simple way to recover delegated resources to other accounts. This PR modifies the UnstakeTab component to select the account from which to recover resources.

This PR also contains the first approach to the AntelopeStore library which will be continued in telos-wallet to finally create its own repo when it is mature enough.

## Test scenarios
- go to [this link](https://deploy-preview-649--obe-staging.netlify.app/)
- log in as `gqydoobuhege` using cleos
- go to Wallet
- press Resources button
- click on Tab "Remove CPU/NET"
- You should see a select component filled with all accounts `gqydoobuhege` has delegated resources to.
- By default `gqydoobuhege` selected
- Change the selected account
- Notice the Available amounts should change too
- If you click on them, the amount should be used
- If you introduce more than available, it should show an error
- Change again the account
- The fields of NET and CPU should be set to 0

## Checklist:
